### ISSUE TITLE
Highlight messages I sent

### ISSUE THREAD

**Issue by Mossop** [vector-im/element-meta#1656]
**Is your suggestion related to a problem? Please describe.**
I find it very difficult to scroll back and find messages I wrote. I'm used to other clients highlighting such messages in a similar way to how mentions are highlighting. I would be good to do this.

**Describe the solution you'd like**
Messages in the room that I sent should be highlighted somehow.

**Describe alternatives you've considered**
None really, not sure what an alternative would be.

**Additional context**
N/A

**Comment by daniellekirkwood** [comment]
We now have chat bubbles in our product, on all platforms. As this satisfies the request, I'll close this out. 

Thanks!

**Comment by Mossop** [comment]
I disagree that chat bubbles satisfies the request as stated. Aside from other usability issues with the chat bubbles appearance \in the case of short messages it makes it even harder to spot my own messages as they are so removed from the rest of the content.

**Comment by daniellekirkwood** [comment]
Ok, thanks for the feedback. I will move this to discussions!
